### ISSUE TITLE
[idea] Knowledge base for contributing to open source projects

### ISSUE THREAD

**Issue by at15** [dyweb/mos#18]
idea issuer: infamous @at15

## Description

For newbie, it's always pretty hard when they want to start contributing to other people/org's projects, the are always some implicit convention in the community, like I came across the golang import path problem today and @gaocegege took it for granted so he didn't know what I am talking about when I asked him. (maybe he is looking at some hot girls, who knows) Since we have some experienced guys i.e. @mrmiywj @gaocegege , it would be great to share their knowledge with others so they can have less pain on figuring out those conventions and focus on the real coding part.

I am not quite sure about the details, like using gitbook or blog etc, so it's a quite open idea.

**Comment by gaocegege** [comment]
Yeah, and I think there should be a section to list some friendly communities for newcomers. Because you know, some communities is critical.

**Comment by gaocegege** [comment]
It's the work mos should do.
/cc @dyweb/mos

**Comment by xplorld** [comment]
would it be some kind of wiki or collection-of-markdown's? It seems more to be a content work than a framework.

**Comment by at15** [comment]
https://github.com/shazow/ssh-chat/wiki/FAQ provides a detail guide of how to contribute to a golang project.

**Comment by ComMouse** [comment]
@gaocegege What about the progress till now?

**Comment by gaocegege** [comment]
Zero :thinking:

**Comment by ComMouse** [comment]
@gaocegege Any plan? 😅

**Comment by gaocegege** [comment]
目前来说，没有 =。=

**Comment by gaocegege** [comment]
ref https://github.com/MunGell/awesome-for-beginners

**Comment by gaocegege** [comment]
I am closing this since it is stale.
